src/libs/Navigation/Navigation.js

```diff
@@ -81,7 +81,7 @@ const getActiveRouteIndex = function (route, index) {
 /**
  * Main navigation method for redirecting to a route.
  * @param {String} route
- * @param {String} type - Type of action to perform. Currently UP is supported.
+ * @param {String} [type] - Type of action to perform. Currently UP is supported.
```

Comment:
we should remove types from JSDoc params

Reply:
@kubabutkiewicz It's js file, not ts